llvm/test/tools/sycl-post-link/device-globals/test_no_metadata_generated.ll

```diff
@@ -0,0 +1,49 @@
+; RUN: sycl-post-link --ir-output-only --device-globals %s -S -o - | FileCheck %s
```

Comment:
This may be used instead of CHECK-NOT comment
```suggestion
; RUN: sycl-post-link --ir-output-only --device-globals %s -S -o - | FileCheck %s --implicit-check-not "!spirv.Decorations"
```

Reply:
Thank you for the suggestion, applied it.